### DISCUSSION TITLE
Cannot delete entire conversation

### DISCUSSION THREAD

**Discussion by mickhub80** [protz/thunderbird-conversations#2004] [Q&A] [answered]
Hi,

on TB v102.14.0 when I click "Delete conversation", only messages in the same folder are deleted i.e. only sent or received messages, depending on the folder of the current message.
Is this expected?

Thanks in advance,
Mick

**Comment by Standard8** [comment, marked as answer]
It is an option:

* Go into the add-on manager (three-bar menu -> Add-ons and Themes).
* Select Extensions on the left.
* Next to Thunderbird Conversations, select the spanner icon (Add-on Options).
* Put a tick in the box next to "Archive/delete behavior"
